### ISSUE TITLE
Enable TagPolicy in LZA for GovCloud

### ISSUE THREAD

**Issue by z-aws** [awslabs/landing-zone-accelerator-on-aws#191]
**Describe the bug**
Attempting to apply an Organization tag policy fails during the Organization action in CodePipeline. The same tag policy attaches to the same OU when done manually in the AWS Console.

**To Reproduce**
Create a JSON tag policy in LZA and define a TaggingPolicyConfig object in the GovCloud LZA configuration. Commit the change and when the CodePipeline reaches the Organization action the following error occurs:

```yaml
taggingPolicies:
  - name: OrganizationPolicy
    description: Organization Tagging Policy
    policy: tagging-policies/org-tag-policy.json
    deploymentTargets:
      organizationalUnits:
        - Infrastructure
```

```json
{
    "tags": {
        "Team": {
            "tag_key": {
                "@@assign": "Team"
            },
            "tag_value": {
                "@@assign": [
                    "sysops",
                    "dbops",
                    "webops"
                ]
            },
            "enforced_for": {
                "@@assign": [
                    "ec2:instance"
                ]
            }
        },
        "Name": {
            "tag_key": {
                "@@assign": "Name"
            },
            "tag_value": {
                "@@assign": []
            }
        },
        "Owner": {
            "tag_key": {
                "@@assign": "Owner"
            },
            "tag_value": {
                "@@assign": []
            }
        },
        "Cluster": {
            "tag_key": {
                "@@assign": "Cluster"
            },
            "tag_value": {
                "@@assign": [
                    "gov",
                    "govdev",
                    "govstg"
                ]
            }
        },
        "Environment": {
            "tag_key": {
                "@@assign": "Environment"
            },
            "tag_value": {
                "@@assign": [
                    "production",
                    "staging",
                    "development",
                    "il4"
                ]
            }
        },
        "CostCenter": {
            "tag_key": {
                "@@assign": "CostCenter"
            },
            "tag_value": {
                "@@assign": [
                    "datacenteroperations"
                ]
            }
        }
    }
}
```

```
AWSAccelerator-OrganizationsStack-<redacted_org_id>-us-gov-east-1 \| 11:12:32 PM \| CREATE_FAILED        \| Custom::EnablePolicyType                          \| enablePolicyTypeTag/Resource/Default (enablePolicyTypeTagB9F93488) Received response status [FAILED] from custom resource. Message returned: Error: Policy Type TAG_POLICY not supported.
--
363 | at pm (/var/task/index.js:32:17362)
364 | at Runtime.handler (/var/task/__entrypoint__.js:1:938)
365 | at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1016:29) (RequestId: 009e19f2-59b8-4104-be58-b0683134dc7f)
366 | new CustomResource (/codebuild/output/src3334/src/s3/00/source/node_modules/aws-cdk-lib/core/lib/custom-resource.js:1:823)
367 | \_ new EnablePolicyType (/codebuild/output/src3334/src/s3/00/source/packages/@aws-accelerator/constructs/lib/aws-organizations/enable-policy-type.ts:88:22)
368 | \_ OrganizationsStack.addTaggingPolicies (/codebuild/output/src3334/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/stacks/organizations-stack.ts:599:35)
369 | \_ new OrganizationsStack (/codebuild/output/src3334/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/stacks/organizations-stack.ts:164:10)
370 | \_ main (/codebuild/output/src3334/src/s3/00/source/packages/@aws-accelerator/accelerator/bin/app.ts:572:35)
371 | \_ processTicksAndRejections (node:internal/process/task_queues:96:5)
372 | \_ async /codebuild/output/src3334/src/s3/00/source/packages/@aws-accelerator/accelerator/bin/app.ts:1017:5
```

**Expected behavior**
I expected my organization tag policy to be successfully enabled and attached to the OU I specified as this works when done manually in the AWS Console.

**Please complete the following information about the solution:**

- [ ] Version: [v1.4.2]
- [ ] Region: [e.g. us-gov-east-1] (The action fails after attempting in this region.
- [ ] Was the solution modified from the version published on this repository? Yes
- [ ] If the answer to the previous question was yes, are the changes available on GitHub? No
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses? No
- [ ] Were there any errors in the CloudWatch Logs? Yes. Only what I see in the CodePipeline output for the Organization action.

**Comment by erwaxler** [comment]
Hello, thank you for creating this issue! I have added this feature request to the team's backlog, I will update this issue accordingly. Thank you again for your interest in the Landing Zone Accelerator on AWS.

**Comment by awsclemj** [comment]
Hello,

This fix was implemented as part of our [v1.5.0 release](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.5.0). Please update at your convenience and you will be unblocked from this error.

Thank you for your support of the LZA solution!

**Comment by jperezpatino** [comment]
After updating to v1.5.2 this still happens:
`
AWSAccelerator-OrganizationsStack-125656955281-us-gov-west-1 \| 15/22 \| 1:41:25 AM \| CREATE_FAILED        \| Custom::EnablePolicyType                          \| enablePolicyTypeTag/Resource/Default (enablePolicyTypeTagB9F93488) Received response status [FAILED] from custom resource. Message returned: Error: Policy Type TAG_POLICY not supported.
--
180 | at vXe (/var/task/index.js:10262:8515)
181 | at Runtime.handler (/var/task/__entrypoint__.js:1:938)
182 | at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1147:29) (RequestId: 24f5871f-f04b-45d0-b95b-34b852a0a0f3)
183 | new CustomResource (/codebuild/output/src3868/src/s3/00/source/node_modules/aws-cdk-lib/core/lib/custom-resource.js:1:823)
184 | \_ new EnablePolicyType (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/constructs/lib/aws-organizations/enable-policy-type.ts:88:22)
185 | \_ OrganizationsStack.addTaggingPolicies (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/stacks/organizations-stack.ts:632:35)
186 | \_ new OrganizationsStack (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/stacks/organizations-stack.ts:177:10)
187 | \_ createOrganizationsStack (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/utils/stack-utils.ts:462:31)
188 | \_ createSingleAccountMultiRegionStacks (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/bin/app.ts:126:29)
189 | \_ main (/codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/bin/app.ts:229:5)
190 | \_ processTicksAndRejections (node:internal/process/task_queues:96:5)
191 | \_ async /codebuild/output/src3868/src/s3/00/source/packages/@aws-accelerator/accelerator/bin/app.ts:240:5

`

`taggingPolicies: 
# This Tag policy is already manually applyed due to LZA solution bug:
# https://github.com/awslabs/landing-zone-accelerator-on-aws/issues/191
  - name: tagging-policy
    description: Organization Tagging Policy
    policy: tagging-policies/org-tag-policy.json
    deploymentTargets:
      organizationalUnits: 
        - Root`
